src/c_kzg_4844.c

```diff
@@ -1796,29 +1725,22 @@ C_KZG_RET load_trusted_setup(
         blst_p2_from_affine(&out->g2_values[i], &g2_affine);
     }
 
-    /* It's the smallest power of 2 >= n1 */
-    unsigned int max_scale = 0;
-    while ((1ULL << max_scale) < n1)
-        max_scale++;
-
     /* Initialize the KZGSettings struct */
```

Comment:
This comment now feels out of place:

```suggestion
    /* Compute roots of unity and permute the G1 trusted setup */
```

Reply:
Agreed. Fixed.